doc/source/install.rst

```diff
@@ -56,8 +56,9 @@ Alternatives for experts
 
 The conda recipe above is intended to give a self-contained and reproducible
 environment with not only dgcode itself, but also any required tools like
-`simplebuild <https://mctools.github.io/simplebuild>`, a Python interpreter and
-all the necessary build tools. For special advanced use-cases, experts might
+`simplebuild
+<https://mctools.github.io/simplebuild>`_, a Python interpreter, and
```

Comment:
Fix is a step in the right direction, but I think the external links have to end with double underscore `__` and not `_`. If there is only one, sphinx generates some extra targets or something like that. In any case, I have tried to use double underscores everywhere.